Casks/f/filefillet.rb

```diff
@@ -0,0 +1,24 @@
+cask "filefillet" do
+  version "2.6.1"
+  sha256 "3719454ca337fd9438641326bf4316889f2b3efe692098670912db708a828316"
+
+  url "https://filefillet-releases.s3.eu-central-1.amazonaws.com/FileFillet_v#{version}.zip",
+      verified: "filefillet-releases.s3.eu-central-1.amazonaws.com/"
```

Comment:
Let's tap into the Sparkle feed for new releases, same as the built-in app updater.

```suggestion
  url https://release.filefillet.com/FileFillet_v#{version}.zip
```